tools/ci_build/windows/extract_nuget_files.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This file is used by Zip-Nuget Packaging NoContribOps Pipeline,Zip-Nuget-Java Packaging Pipeline
+
+# Re-construct a build directory that contains binaries from all the different platforms we're including
+# in the native ORT nuget package
+$nuget_artifacts_dir = "$Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo\nuget-artifacts"
+New-Item -Path $nuget_artifacts_dir -ItemType directory
+
+## .zip files
+# unzip directly
+Get-ChildItem $Env:BUILD_BINARIESDIRECTORY\nuget-artifact -Filter *.zip |
+Foreach-Object {
+ $cmd = "7z.exe x $($_.FullName) -y -o$nuget_artifacts_dir"
+ Write-Output $cmd
+ Invoke-Expression -Command $cmd
+}
+
+## .tgz files
+# first extract the tar file from the tgz
+Get-ChildItem $Env:BUILD_BINARIESDIRECTORY\nuget-artifact -Filter *.tgz |
+Foreach-Object {
+ $cmd = "7z.exe x $($_.FullName) -y -o$Env:BUILD_BINARIESDIRECTORY\nuget-artifact"
+ Write-Output $cmd
+ Invoke-Expression -Command $cmd
+}
+
+# now extract the actual folder structure from the tar file to the build dir
+Get-ChildItem $Env:BUILD_BINARIESDIRECTORY\nuget-artifact -Filter *.tar |
+Foreach-Object {
+ $cmd = "7z.exe x $($_.FullName) -y -o$nuget_artifacts_dir"
+ Write-Output $cmd
+ Invoke-Expression -Command $cmd
+}
+
+# copy android AAR.
+# for full build of onnxruntime-extensions Android AAR, there should only be one .aar file
+# called onnxruntime-extensions-android-x.y.z.aar but sanity check that
```

Comment:
I don't think this check is relevant/needed here. It was for ORT where we potentially had 'full' and 'mobile' AAR files.